app/app/ .env.local

```diff
@@ -0,0 +1,2 @@
+
+NEXT_PUBLIC_API_BASE_URL=http://localhost:3001
```

Comment:
add this to .gitignore file and add a `.env.local.example` instead which can be on Github.

Reply:
This file is at the wrong location, it should be at `app/.env.local`